packages/lit-dev-content/site/tutorials/content/svg-templates/05.md

```diff
@@ -0,0 +1,10 @@
+05.md
+
+// theme the repeating tile pattern using css properties
+
+// font-size
+// font color
+// stroke color
+// fill color
+
+// text
```

Comment:
add a congrats you're done page with a what's next / further reading section with a list of links to docs and mdn links that would logically follow. The user might not be knowledgable enough about web dev to know what topics to follow up with.

Reply:
sg